components/nimbus/src/matcher.rs

```diff
@@ -67,4 +69,6 @@ pub struct AppContext {
     pub os_version: Option<String>,
     pub android_sdk_version: Option<String>,
     pub debug_tag: Option<String>,
+    #[serde(flatten)]
+    pub specific_context: Option<HashMap<String, String>>,
```

Comment:
Just to be 1000% sure, the fact that this is optional, it doesn't break backward compatibility right?

Reply:
Yeah, as long as this is optional, then it shouldn't fail serializing/deserializing if it isn't there.